docs/ABOUT.md

```diff
@@ -0,0 +1,9 @@
+[C++](https://en.wikipedia.org/wiki/C%2B%2B) (pronounced `cee plus plus`) is a widely used, compiled, multi-paradigm language that extends
+the basic features found in C. Originally created by Bjarne Stroustrup at Bell Labs, it is often used in places where speed and very fine
+grained control over a program's execution are needed. It is most often found in use as a systems language, and is also great for
+situations where resources are significantly constrained. The standard for C++ is maintained by the International Organization for
+Standardization (ISO), and the current version of C++ as of December 2014 is C++14 (named not as the 14th version of C++, but rather
+signifying that the standard was ratified in 2014).
+
+_Help us explain this better! File a GitHub issue at https://github.com/exercism/xcpp/issues if you have suggestions, or submit a
```

Comment:
This bit is included in the website for all the languages--it doesn't need to be included here.